util/text.py

```diff
@@ -5,28 +5,35 @@
 from six.moves import range
 from functools import reduce
 
-# Constants
-SPACE_TOKEN = '<space>'
-SPACE_INDEX = 0
-FIRST_INDEX = ord('a') - 1  # 0 is reserved to space
-
-def text_to_char_array(original):
+class Alphabet(object):
+    def __init__(self, config_file):
+        self._label_to_str = []
+        self._str_to_label = {}
+        self._size = 0
+        with open(config_file, 'r') as fin:
+            for line in fin:
+                self._label_to_str += line[:-1] # remove the line ending
```

Comment:
What about the final line which may not have a line ending.

Reply:
I'll add a comment mentioning that the file must end with a newline :P